examples/clients/Skeleton.c

```diff
@@ -0,0 +1,54 @@
+/** @file
+    @brief Implementation
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Internal Includes
+#include <osvr/ClientKit/ContextC.h>
+#include <osvr/ClientKit/InterfaceC.h>
+#include <osvr/ClientKit/InterfaceStateC.h>
+
+// Library/third-party includes
+
+// Standard includes
+#include <stdio.h>
+
+int main() {
+    OSVR_ClientContext ctx =
+        osvrClientInit("com.osvr.exampleclients.SkeletonCallback", 0);
+
+    OSVR_ClientInterface leftHand = NULL;
+    OSVR_ClientInterface rightHand = NULL;
+
+    osvrClientGetInterface(ctx, "/me/hands/left", &leftHand);
+    osvrClientGetInterface(ctx, "/me/hands/right", &rightHand);
+
```

Comment:
This needs some actual subscriptions to skeleton report callbacks, preferably one that outputs a report to the console.